text-ui-test/runtest.sh

```diff
@@ -13,7 +13,7 @@ then
 fi
 
 # compile the code into the bin folder, terminates if error occurred
-if ! javac -cp ../src -Xlint:none -d ../bin ../src/main/java/Duke.java
+if ! javac -cp ../src -Xlint:none -d ../bin ../src/main/java/*.java ../src/main/java/cmd/*.java ../src/main/java/command/*.java ../src/main/java/misc/*.java ../src/main/java/task/*.java
```

Comment:
Perhaps globstar could be enabled, so that this script doesn't have to change every time a new package is added?

EDIT: I realize this is out of the scope of the review but ya

Reply:
Nice comment, I've learnt something new today!
I went ahead and attempted to adapt globstar in runtest.bat, but it seems like there are problems using ** and *.java together in batch script. Nevertheless, i will find time to implement and test it on runtest.sh. Thanks for the suggestion!

For reference, in runtest.bat, i tried
javac  -cp ..\src -Xlint:none -d ..\bin ..\src\main\java\duke\Main.java ..\src\main\java\duke\**\*.java // Illegal char '*'
javac  -cp ..\src -Xlint:none -d ..\bin ..\src\main\java\duke\**\*.java // Illegal char '*'
javac  -cp ..\src -Xlint:none -d ..\bin ..\src\main\java\duke\** // works, but dir only

Do let me know if i missed anything.

Reply:
Update: Globstar seems to work on shell script, but sunfire and compute clusters doesn't seem to have java11

Reply:
Globstar is a Bash feature, it does not exist in batch scripts. Sunfire is ancient and really just used for jumping. There might be several nodes with Java 11. Perhaps `xgpe1`?

Reply:
Yup, managed to verify on xgpe1. Tyvm